components/ActionModals/ClaimModal.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <ActionModal
+    id="modal-withdraw-rewards"
+    ref="actionModal"
+    :transaction-data="transactionData"
+    :notify-message="notifyMessage"
+    :selected-denom="feeDenom"
+    title="Claim Rewards"
+    class="modal-withdraw-rewards"
+    submission-error-prefix="Withdrawal failed"
+    feature-flag="claim_rewards"
+    :transaction-type="lunieMessageTypes.CLAIM_REWARDS"
+    :rewards="rewards"
+    :disable="validatorsWithRewards"
+  >
+    <span class="form-message withdraw-limit">
```

Comment:
let's do this only if the user is logged in with Ledger

Reply:
we also don't need to restrict to `top5Validators` if it is not logged in with Ledger

Reply:
That is what I meant